packages/meditrak-app-server/src/routes/sync/PushChangesRoute/types.ts

```diff
@@ -1 +1 @@
-export type SurveyResponseObject = Record<string, unknown>;
+export type SurveyResponseObject = Record<string, any>;
```

Comment:
Would love if we could still get things working with `unknown` rather than `any`. `any` is kind of a bad practice in typescript as it allows for skipping validation entirely... Happy to do a bit of a pair on it if you want?

Also, quite OCD, but can you add the copyright to the top of the file 😅